functions/system.bash

```diff
@@ -213,6 +213,33 @@ misc_system_settings() {
   echo "OK"
 }
 
+### change swap (int size in MB)
```

Comment:
Use 2 `#` i.e. `## change swap (int size in MB)`.

Reply:
ok